src/utility/index.jsx

```diff
@@ -124,30 +118,67 @@ export const getOrdinal = (n) => {
 
 const getSubtext = (row) => {
   if (row.match_id && row.player_slot !== undefined) {
-    // TODO localize
-    return isRadiant(row.player_slot) ? 'Radiant' : 'Dire';
+    return isRadiant(row.player_slot) ? strings.radiant : strings.dire;
   }
   if (row.last_played) {
     return <FromNowTooltip timestamp={row.last_played} />;
   }
+  if ((row.name && row.value) !== undefined) {
+    return <FromNowTooltip timestamp={row.start_time} />;
+  }
   return null;
 };
 
+const percentColor = (pct) => {
```

Comment:
Hey, we can use dota emoticons instead if letters or gm icons. I'll do it after a while 😺